vault/logical_system.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"crypto/sha256"
 	"crypto/sha512"
+	"crypto/subtle"
```

Comment:
Meta comment: I think it might be nice to move the new stuff into logical_system_raft. This file is huge enough as is...